### PR TITLE
[CloudPC] Documentation update for new API - GetProvisionedCloudPCs

### PR DESCRIPTION
**Pull request by SleepIsImportant**

### CODE REVIEW

File: api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+<!-- {
+  "blockType": "request",
+  "name": "cloudpc.getProvisionedCloudPCs"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/cloudPCs/getProvisionedCloudPCs(groupId='30d0e128-de93-41dc-89ec-33d84bb662a0',servicePlanId='9ecf691d-8b82-46cb-b254-cd061b2c02fb')
+```
+
+# [C#](#tab/csharp)
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md:77]: Remove line 77 to line  99. No need to add this.

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]
Please add changelog to Microsoft.CloudManageDesktop.json.
Please update toc.yml to include the cloudpc-getprovisionedcloudpcs api entry point https://learn.microsoft.com/en-us/graph/api/resources/cloudpcreviewstatus?view=graph-rest-beta.
image

**Comment by AshleyYangSZ** [comment]
apidoctor build failed, please help resolve.

File: api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+
+## Request parameter
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|groupId|Guid|The unique identifier for Azure Active Directory (AAD) user group.|
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md:49]: Update Guid to String. Same for below parameter.

File: api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+If successful, this method returns a `200 OK` response code and a [cloudPC](../resources/cloudPC.md) collection object in the response body.
+
+## Examples
+
+### Request
+
+The following is an example of a request.
+
+
+# [HTTP](#tab/http)
```
Comment (RuiHou105) [inline api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md:67]: Remove this extra line.

File: api-reference/beta/toc.yml [hunk 4]
```diff
@@ -10623,6 +10623,8 @@ items:
             href: api/cloudpc-troubleshoot.md
           - name: Retry partner agent installation
             href: api/cloudpc-retrypartneragentinstallation.md
+          - name: Get provisioned cloudPCs by groupId and servicePlanId
```
Comment (RuiHou105) [inline api-reference/beta/toc.yml:10626]: Please move this one after `- name: Get launch info for user`, as all functions/actions should be ordered alphabetically.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@SleepIsImportant  Please also update the changelog to describe your change.

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 5]
```diff
@@ -1,5 +1,23 @@
 {
     "changelog": [
+        {
+            "ChangeList": [
+                {
+                    "Id": "b44e2995-4ab6-4714-85b6-6e3119d4c4cc",
+                    "ApiChange": "Method",
+                    "ChangedApiName": "getProvisionedCludPcs",
```
Comment (paulodero) [inline changelog/Microsoft.CloudManagedDesktop.json:8]: Should be `getprovisionedcloudpcs`?

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 6]
```diff
@@ -1,5 +1,23 @@
 {
     "changelog": [
+        {
+            "ChangeList": [
+                {
+                    "Id": "b44e2995-4ab6-4714-85b6-6e3119d4c4cc",
+                    "ApiChange": "Method",
+                    "ChangedApiName": "getProvisionedCloudPcs",
+                    "ChangeType": "Addition",
+                    "Description": "Added the [getProvisionedCloudPcs](https://learn.microsoft.com/en-us/graph/api/cloudpc?view=graph-rest-beta) method to the [cloudPC](https://learn.microsoft.com/en-us/graph/api/resources/cloudpc?view=graph-rest-beta) for end-user.",
```
Comment (AshleyYangSZ) [inline changelog/Microsoft.CloudManagedDesktop.json:10]: The link is not correct, it should link to API file not cloudpc resource file
`[getProvisionedCloudPcs](https://learn.microsoft.com/en-us/graph/api/cloudpc?view=graph-rest-beta) method`

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 6 above)
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:10]:
```suggestion
                    "Description": "Added the [getProvisionedCloudPcs](https://learn.microsoft.com/en-us/graph/api/cloudpc-getprovisionedcloudpcs?view=graph-rest-beta) method to the [cloudPC](https://learn.microsoft.com/en-us/graph/api/resources/cloudpc?view=graph-rest-beta) for end-user.",
```

File: api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md (hunk 2 above)
Comment (Danielabom) [reply api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md:49]: @SleepIsImportant, I do not see Rui's comments applied. Please review.

**Review by Danielabom — CHANGES_REQUESTED** [review]
@SleepIsImportant, thank you for working on this PR. Please review my comments.

**Review by Danielabom — APPROVED** [review]
PR approved.

File: api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md (hunk 2 above)
Comment (Danielabom) [reply api-reference/beta/api/cloudpc-getprovisionedcloudpcs.md:49]: @SleepIsImportant @RuiHou105, reposting: I am confused about Rui's comment. Should the type be updated or not? Please make sure all technical reviewer comments have been resolved before moving to the content review. Thanks!

**Comment by SleepIsImportant** [comment]
> 
fixed changedLog thank you

**Comment by SleepIsImportant** [comment]
> @SleepIsImportant Please also update the changelog to describe your change.

Fixed, thank you

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 7]
```diff
@@ -3,19 +3,19 @@
...
                 {
-                    "Id": "bebbf84c-73de-4070-b44f-c670c25c4d4f",
-                    "ApiChange": "Parameter",
-                    "ChangedApiName": "LIST",
+                    "Id": "b44e2995-4ab6-4714-85b6-6e3119d4c4cc",
+                    "ApiChange": "Method",
+                    "ChangedApiName": "getProvisionedCloudPcs",
                     "ChangeType": "Addition",
-                    "Description": "Enabled the `top` query parameter for the [LIST](https://learn.microsoft.com/en-us/graph/api/virtualendpoint-list-cloudpcs?view=graph-rest-beta) operation of the [cloudPC](https://learn.microsoft.com/en-us/graph/api/resources/cloudpc?view=graph-rest-beta) resource.",
+                    "Description": "Added the [getProvisionedCloudPcs](https://learn.microsoft.com/en-us/graph/api/cloudpc-getprovisionedcloudpcs?view=graph-rest-beta) method to the [cloudPC](https://learn.microsoft.com/en-us/graph/api/resources/cloudpc?view=graph-rest-beta) for end-user.",
```
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:10]:
```suggestion
                    "Description": "Added the [getProvisionedCloudPcs](https://learn.microsoft.com/en-us/graph/api/cloudpc-getprovisionedcloudpcs?view=graph-rest-beta) method to the [cloudPC](https://learn.microsoft.com/en-us/graph/api/resources/cloudpc?view=graph-rest-beta) resource.",
```

File: api-reference/beta/toc.yml [hunk 8]
```diff
@@ -10568,6 +10568,8 @@ items:
             href: api/cloudpc-getcloudpclaunchinfo.md
           - name: Get shift work access state
             href: api/cloudpc-getshiftworkcloudpcaccessstate.md
+          - name: Get provisioned cloudPCs by groupId and servicePlanId
```
Comment (Danielabom) [inline api-reference/beta/toc.yml:10571]:
```suggestion
          - name: Get provisioned Cloud PCs by group Id and service plan id
```

**Review by Danielabom — APPROVED** [review]
PR approved.

**Comment by Danielabom** [comment]
'@SleepIsImportant, please add the "ready-to-merge" label if you are ready to publish.

**Pull request merged**